### DISCUSSION TITLE
greylisting module

### DISCUSSION THREAD

**Discussion by srsr** [rspamd/rspamd#4119] [Q&A] [answered]
Hello to all,
Just a small question, is it normal that the greylisting module is activated by default?
In the documentation it is indicated that you have to do before this :

> To enable the module with default settings you need to define at least [redis](https://rspamd.com/doc/configuration/redis.html) servers to store greylisting data:
> 
> local.d/greylist.conf
> servers = "127.0.0.1:6379";
> 
> Adding servers to store greylisting data enables greylisting in Rspamd.

I wonder if it is a new setting, a non updated documentation or if I missing something?

Thanks

Samuel

**Comment by moisseev** [comment, marked as answer]
The greylisting module definitely cannot work without Redis. It is likely that you have enabled Redis **globally** in redis.conf

See https://rspamd.com/doc/configuration/redis.html#redis-setup :
`Setting Redis options for each individual module might be simplified by using of common redis section (for example, by defining it in /etc/rspamd/local.d/redis.conf):`

**Comment by srsr** [comment]
Thanks moisseev,
this is my case indeed.
